lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -1462,6 +1464,87 @@ static bool DumpModuleSymbolFile(Stream &strm, Module *module) {
   return false;
 }
 
+static bool GetSeparateDebugInfoList(StructuredData::Array &list,
+                                     Module *module) {
+  if (module) {
+    if (SymbolFile *symbol_file = module->GetSymbolFile(true)) {
```

Comment:
Merge into one line to reduce indention?
```
if (module && SymbolFile *symbol_file = module->GetSymbolFile(true)) {
}
```

Reply:
That's not valid syntax, and I'm not sure how to use a single `if` statement to capture the meaning. I could write by declaring `symbol_file` outside of the `if` statement condition, but I think that's unnecessarily verbose. Besides, 10 lines up we do the same thing.